packages/next/shared/lib/loadable.js

```diff
@@ -154,7 +156,10 @@ function createLoadableComponent(loadFn, options) {
           retry: subscription.retry,
         })
       } else if (state.loaded) {
-        return React.createElement(resolve(state.loaded), props)
+        return React.createElement(resolve(state.loaded), {
+          ...props,
+          ref: innerRef,
+        })
```

Comment:
IMHO this change will not work. `resolve(state.loaded)` can/will (and most likely) be a functional component, where it doesn't accept refs.

Reply:
Oh good catch! I ran into this issue with a non-functional component.